### PR TITLE
created latest notice section

### PR DESCRIPTION
**Pull request by ChathukaViduranga**

### CODE REVIEW

File: src/components/client/landing/latestNotice/LatestNotice.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React, { useEffect } from "react"
+import "./latestNotice.css"
+import notice1 from './../../../../assets/images/notice/notice1.jpeg'
```
Comment (senuravihanjayadeva) [inline src/components/client/landing/latestNotice/LatestNotice.jsx]: Dont use jpeg. Please covert images to webp. Do the same thing for others

File: src/components/client/landing/latestNotice/LatestNotice.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <div className="latestNotices">
+
+
+    <div className="slideshow-container">
+       <h2>LATEST NOTICES </h2>
+ </div>
+ <div className='carousel'>
+        <div className='carousel_track-container'>
+          <ul className='carousel__track'>
+            {images.map((image, index) => (
```
Comment (senuravihanjayadeva) [inline src/components/client/landing/latestNotice/LatestNotice.jsx]: First check images exists or not

File: src/components/client/landing/latestNotice/LatestNotice.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+              <li key={index} className='carousel_slide current-slide'>
+                <img className='carousel-images' src={image} alt='notice' />
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div className='carousel__nav'>
+          {images.map((_, index) => (
```
Comment (senuravihanjayadeva) [inline src/components/client/landing/latestNotice/LatestNotice.jsx]: check images exists or not

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]
Please maintain formatting using prettier.
Change the target branch to feature/dev

File: src/components/client/landing/latestNotice/LatestNotice.jsx (hunk 3 above)
Comment (ChathukaViduranga) [reply src/components/client/landing/latestNotice/LatestNotice.jsx]: ok. I'll do it

**Comment by uihalagedara** [comment]
upload a screenshot of the implemented component as well

**Comment by ChathukaViduranga** [comment]
Screenshot 2566-03-15 at 14 56 03

**Comment by ChathukaViduranga** [comment]
WhatsApp Image 2023-03-15 at 15 03 38

**Pull request merged**